### PR TITLE
change paragraph

### LINKED ISSUE #200: Update Challenge Page Description
The current description is based on the old Stride For Education event rather than how we handle challenges now. 
The user no longer chooses a challenge, rather challenges are used as a reference point on their dashboard to visualise their mileage. Their challenge progress resets after 14 days but we'll keep the daily kms based on 12 days to give them some leeway if they forget to log. The challenge page is just there to give them information.

### PR DESCRIPTION
**Pull request by Alex-Hawking**
## Change Summary

Re-wrote the paragraph

### Change Form

**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

Not sure if the description should have 12 or 14 days? I've put 14 rn

# Related Issue

- Resolve #200

### CODE REVIEW

**Review by RadioArsenic — CHANGES_REQUESTED** [review]
Say it's over a 12 day period so the daily kms make sense, but try to mention somehow that progress resets after 14

**Comment by RadioArsenic** [comment]
Also remove the added favicon

File: client/src/views/ChallengeView.vue [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
       <h2 style="margin-bottom: 1vh" class="text-subtitle-1 text-md-h6 font-weight-bold">
-        Choose your challenge to complete over the 14 days of the challenge
+        Log your daily kilometers over the 12-day period, and watch your progress on the personal
+        dashboard. Challenges serve as reference points for your achievements, providing valuable
+        insights into your journey. The challenge progress resets every 14 days giving you some extra time if you
+        forget to record your journey.
       </h2>
       <h2 class="text-subtitle-1 text-md-h6">
-        (Don't worry if you can't hit your daily target, just do your best!)
+        (Don't worry if you can't complete all the challenges, just do you best!)
```
Comment (ChantelleKerr) [inline client/src/views/ChallengeView.vue:13]: should be "just do your best!"

**Review by RadioArsenic — CHANGES_REQUESTED** [review]
"and watch your progress on your personal dashboard"

Should be your dash not the dash

**Review by RadioArsenic — APPROVED** [review]
Looks good!

**Pull request merged**